artifacts/openapi.yaml

```diff
@@ -598,206 +610,1872 @@ components:
-          default: do_not_include_as
-        advanced:
-          $ref: '#/components/schemas/Device.BgpAdvanced'
-        capability:
-          $ref: '#/components/schemas/Device.BgpCapability'
-        sr_te_policies:
-          description: |-
-            Segment routing/traffic engineering policies
-          type: array
-          items:
-            $ref: '#/components/schemas/Device.BgpSrTePolicy'
-        bgpv4_routes:
-          description: |-
-            Emulated BGPv4 routes
-          type: array
-          items:
-            $ref: '#/components/schemas/Device.Bgpv4Route'
-        bgpv6_routes:
+          format: ipv6
+        prefix:
           description: |-
-            Emulated BGPv6 routes
-          type: array
-          items:
-            $ref: '#/components/schemas/Device.Bgpv6Route'
+            The network prefix.
+          type: integer
+          minimum: 1
+          maximum: 128
+          default: 64
         name:
           description: |-
             Globally unique name of an object. It also serves as the primary key for arrays of objects.
           type: string
           pattern: ^[\sa-zA-Z0-9-_()><\[\]]+$
-        active:
-          description: |-
-            If enabled means that this part of the configuration including any active 'children' nodes will be advertised to peer.  If disabled, this means that though config is present, it is not taking any part of the test but can be activated at run-time to advertise just this part of the configuration to the peer.
-          type: boolean
-          default: true
-    Device.BgpAdvanced:
       x-status: under-review
+    Interface.LoopbackIpv6:
       description: |-
-        Container for BGP advanced settings.
+        An IPv6 loopback interface
       type: object
+      required:
+      - interface_name
       properties:
-        hold_time_interval:
-          description: |-
-            Number of seconds the sender proposes for the value of the Hold Timer
-          type: integer
-          default: 90
-        keep_alive_interval:
-          description: |-
-            Number of seconds between transmissions of Keep Alive messages by router
-          type: integer
-          default: 30
-        update_interval:
+        interface_name:
+          description: "The unique name of ethernet object behind which this loopback\
+            \ \n interface will be created.\n\nx-constraint:\n- /components/schemas/Interface.Ethernet/properties/name\n"
+          type: string
+          x-constraint:
+          - /components/schemas/Interface.Ethernet/properties/name
+        address:
           description: |-
-            The time interval at which UPDATE messages are sent to the DUT, expressed as the number of milliseconds between UPDATE messages.
-          type: integer
-          default: 0
-        time_to_live:
+            The Loopback IPv6 address
+          type: string
+          format: ipv6
+          default: ::0
```

Comment:
again, is this default a good one that will work 99% of the time or any other default that will work 99% of time?  If not, make this a required property.